content/uc-doc/contributors/style_guide.md

```diff
@@ -6,363 +6,403 @@ title: Style Guide
 
 Example:
 
-    pep8 --ignore=E501 wazo_calld
+```shell
+tox -e black
+```
 
 When possible, avoid using backslashes to separate lines.
 
 Bad Example:
 
-    user = session.query(User).filter(User.firstname == firstname)\
-                              .filter(User.lastname == lastname)\
-                              .filter(User.number == number)\
-                              .all()
+```python
+user = session.query(User).filter(User.firstname == firstname)\
+                          .filter(User.lastname == lastname)\
+                          .filter(User.number == number)\
+                          .all()
+```
 
 Good Example:
 
-    user = (session.query(User).filter(User.firstname == firstname)
-                               .filter(User.lastname == lastname)
-                               .filter(User.number == number)
-                               .all())
+```python
+user = (session.query(User).filter(User.firstname == firstname)
+                           .filter(User.lastname == lastname)
+                           .filter(User.number == number)
+                           .all())
+```
 
 ### Strings {#strings}
 
-Avoid using the [+]{.title-ref} operator for concatenating strings. Use
-string interpolation instead.
+Avoid using the `+` operator for concatenating strings. Use format string instead.
 
 Bad Example:
 
-    phone_interface = "SIP" + "/" + username + "-" + password
+```
+phone_interface = 'SIP' + '/' + username + '-' + password
+```
 
 Good Example:
 
-    phone_interface = "SIP/%s-%s" % (username, password)
+```
+phone_interface = f'SIP/{username}-{password}'
+```
 
 ### Comments {#comments}
 
-Redundant comments should be avoided. Instead, effort should be put on
-making the code clearer.
+Redundant comments should be avoided. Instead, effort should be put on making the code clearer.
 
 Bad Example:
 
-    #Add the meeting to the calendar only if it was created on a week day
-    #(monday to friday)
-    if meeting.day > 0 and meeting.day < 7:
-        calendar.add(meeting)
+```python
+#Add the meeting to the calendar only if it was created on a week day
+#(monday to friday)
+if meeting.day > 0 and meeting.day < 7:
+    calendar.add(meeting)
+```
 
 Good Example:
 
-    def created_on_week_day(meeting):
-        return meeting.day > 0 and meeting.day < 7
+```python
+def created_on_week_day(meeting):
+    return meeting.day > 0 and meeting.day < 7
 
-    if created_on_week_day(meeting):
-        calendar.add(meeting)
+if created_on_week_day(meeting):
+    calendar.add(meeting)
+```
 
 ### Conditions {#conditions}
 
-Avoid using parenthesis around if statements, unless the statement
-expands on multiple lines or you need to nest your conditions.
+Avoid using parenthesis around if statements, unless the statement expands on multiple lines or you
+need to nest your conditions.
 
 Bad Examples:
 
-    if(x == 3):
-        print "condition is true"
+```python
+if(x == 3):
+    print "condition is true"
```

Comment:
python2 syntax

Reply:
Feel free to improve this page/commit as you want :pray: